### PR TITLE
[TS] LPS-131525

### PR DESCRIPTION
**Pull request by ChrisKian**
From @balazssk:

> [VerifyGroup](https://github.com/liferay/liferay-portal/blob/8625f5ed6c4ca75a6520153cab32d121d75fa2bb/portal-impl/src/com/liferay/portal/verify/VerifyGroup.java#L233) uses Collection.contains to filter for userGroupGroupRoles that are present in the Staging group but not in the Live group. This however won't work with the default Object#equals, so I'd like to add this implementation.
> 
> The solution is based on https://github.com/liferay/liferay-portal/commit/35e3d3329ccba0262adb03fc4b81dc8c75618e27#diff-e89f78642ff2532ec9fbd54692f8cf0a1168886d7c7337694925e3299d9f78b9 .
> 
> Can you please review?
> https://issues.liferay.com/browse/LPS-131525 

This issue is unknown on how to reproduce cleanly, however the solution looks safe and is very similar to several other model implementations.

### CODE REVIEW

File: portal-impl/src/com/liferay/portal/model/impl/UserGroupGroupRoleImpl.java [hunk 1]
```diff
@@ -28,6 +30,28 @@
...
+
+		if (!(object instanceof UserGroupGroupRole)) {
+			return false;
+		}
+
+		UserGroupGroupRole userGroupGroupRole = (UserGroupGroupRole)object;
+
+		if ((getUserGroupId() == userGroupGroupRole.getUserGroupId()) &&
+			(getGroupId() == userGroupGroupRole.getGroupId()) &&
+			(getRoleId() == userGroupGroupRole.getRoleId())) {
```
Comment (drewbrokke) [inline portal-impl/src/com/liferay/portal/model/impl/UserGroupGroupRoleImpl.java:47]: nit: these could be sorted alphabetically

**Review by drewbrokke — APPROVED** [review]
LGTM 👍  Only the one nitpick on sorting the expressions in the `if` statement, it's ok if we leave it.

**Comment by drewbrokke** [comment]
@pei-jung if this looks good to you, go ahead and forward.

**Comment by pei-jung** [comment]
ci:forward

**Pull request closed without merging**